### ISSUE TITLE
CSS file is missing in the main section of bower.json

### ISSUE THREAD

**Issue by edmondchui** [guylabs/ion-autocomplete#6]
As a result, running ```grunt wiredep``` does not inject the CSS to the HTML together with the JS file.

**Comment by guylabs** [comment]
Hi,

could you maybe test if it now works with `grunt wiredep` ?

Thanks and regards,

Guy

**Comment by edmondchui** [comment]
Hi Guy,

I can't test your change until you bump the version number to 0.2.1.

Thanks,
Edmond

**Comment by guylabs** [comment]
Hi Edmond,

can you try to install the `ion-autocomplete` with the following command:

```bash
bower install --save-dev 'https://github.com/guylabs/ion-autocomplete.git#master'
```

This gets the dependency with the latest master changes, such that you don't need a new version ;).

Thanks and regards,

Guy

**Comment by edmondchui** [comment]
Hi Guy,

Yup, the latest master changes work with ```grunt wiredep```. Thanks.!

I'll close this issue when the official 0.2.1 version is out.

Thanks,
Edmond

**Comment by guylabs** [comment]
Hi Edmond,

I just released 0.2.1 -> https://github.com/guylabs/ion-autocomplete/releases/tag/v0.2.1

Regards,

Guy

**Comment by edmondchui** [comment]
Thanks Guy. I just get the v0.2.1 release and it works like a charm!
